source/install/config-mattermost-server.rst

```diff
@@ -1,17 +1,18 @@
-Configuring Mattermost Server
+Configuring Mattermost server
```

Comment:
This will appear with lower capitals on the left-hand side navigation page

Reply:
Good catch. This is the same issue that occurred with the integrations doc. This change will be reverted.